docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Could you add <code>!pip install -q --no-deps tensorflow-addons~=0.6</code> we'll use this notebook as the test for the callback so the notebook will need to be able to execute</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
got it!

Reply:
updated in the latest patch!